completions/zsh/_brew

```diff
@@ -111,6 +111,7 @@ __brew_common_commands() {
     'edit:edit a formula'
     'fetch:download formula resources to the cache'
     'formula:the path for a formula'
+		'formulae:show a sorted list of installable formulae'
```

Comment:
Perhaps, change installable to available (?).

Reply:
Sounds good 👍🏻

Reply:
Changed!